src/args/arg.rs

```diff
@@ -3311,6 +3312,16 @@ impl<'a, 'b> Arg<'a, 'b> {
         self
     }
 
+    /// Specifies that if the value is not passed in as an argument, that it should be retrieved
+    /// from the environment if available. If it is not present in the environment, then default
+    /// rules will apply.
+    pub fn from_env<K: AsRef<OsStr>>(mut self, name: K) -> Self {
```

Comment:
It's nitpicky, but I'd rather use a name such as simply `env` or `env_var` so as not to confuse anyone. Typically, `from_*` are constructors which create the `Arg` from something, in a similar manner to `From<T>` traits.

Reply:
yeah, I was thinking about that. The linked issue has `env_fallback`. Personally I'm fine with just `env`